packages/nimble-components/src/select/tests/select-opened-matrix.stories.ts

```diff
@@ -27,8 +28,8 @@ const component = ([
     positionStyle
 ]: PositionState): ViewTemplate => html`
     <${selectTag} open position="${() => position}" style="${() => positionStyle}">
```

Comment:
Could you add text to the stories that describes the new slots, similar to what we have in other components like the button, text field, and tree item?

I wonder if we should also include usage guidance. For example, I'd propose that apps should typically set the start slot for all items or no items. Setting it for just some items causes the alignment of the items to be inconsistent, making it hard to scan their text. Or if this is a valid use case, maybe we need guidance to put some fixed padding in the slot if some items won't have icons.

We can ask @NIbokeefe about it at standup today.

Reply:
@m-akinc Moving to draft because there was some additional discussion to resolve about the direction.